cmd_generate.go

```diff
@@ -257,6 +257,39 @@ func varsToArgs(v vars) string {
 var helperFuncs = template.FuncMap{
 	"up":   strings.ToUpper,
 	"down": strings.ToLower,
+	"upFirst": upFirst,
+	"downFirst": downFirst,
+	"replace": strings.ReplaceAll,
+}
+
+func upFirst(s string) string {
+	var res string
+
+	// iterate over runes
+	for i, c := range s {
```

Comment:
```go
package main

import (
        "strings"
        "testing"
        "unicode"
)

func iterative(s string) string {
        var res string

        // iterate over runes
        for i, c := range s {
                if i == 0 {
                        res += strings.ToLower(string(c))
                } else {
                        res += string(c)
                }
        }

        return res
}

func slicing(s string) string {
        for i, v := range s {
                return string(unicode.ToLower(v)) + s[i+1:]
        }
        return ""
}

func BenchmarkIteratinve(b *testing.B) {
        for i := 0; i < b.N; i++ {
                iterative("Hello world")
        }
}

func BenchmarkSlicing(b *testing.B) {
        for i := 0; i < b.N; i++ {
                slicing("Hello world")
        }
}

```

go test -bench=.
goos: darwin
goarch: amd64
BenchmarkIteratinve-16           2227485               505 ns/op
BenchmarkSlicing-16             26628272                42.0 ns/op
PASS

Reply:
`s[i+1:]` will not work for some unicode values, but I took your code as example, now it looks much better, thanks